src/pages/EditRequestMerchantPage.js

```diff
@@ -27,7 +27,7 @@ function EditRequestMerchantPage({defaultMerchant, onSubmit, isPolicyExpenseChat
     const styles = useThemeStyles();
     const {translate} = useLocalize();
     const merchantInputRef = useRef(null);
-    const isEmptyMerchant = defaultMerchant === '' || defaultMerchant === CONST.TRANSACTION.UNKNOWN_MERCHANT || defaultMerchant === CONST.TRANSACTION.PARTIAL_TRANSACTION_MERCHANT;
+    const isEmptyMerchant = defaultMerchant === '' || defaultMerchant === CONST.TRANSACTION.PARTIAL_TRANSACTION_MERCHANT;
```

Comment:
Can we remove this const as well from here ?
https://github.com/Expensify/App/blob/2acbd813a307a028fd574ba1132e06f9a2da397e/src/CONST.ts#L750

I don't see its usage anywhere else

Reply:
I was thinking about this but its still a special term which we might use in future so I would leave it in for now